Description: Fix spelling errors.
 * lenght      -> length
 * aditionally -> additionally
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://trac.osgeo.org/grass/ticket/3476
Applied-Upstream: https://trac.osgeo.org/grass/changeset/72063

--- a/imagery/i.ortho.photo/i.ortho.photo/i.ortho.photo.html
+++ b/imagery/i.ortho.photo/i.ortho.photo/i.ortho.photo.html
@@ -54,7 +54,7 @@ Five groups of input parameters are requ
 <ul> 
 <li>Aerial image (images),</li>
 <li>Exposure and characteristics of the camera, i.e. its coordinates in
-target coordinate system and height above sea level, focal lenght, yaw, 
+target coordinate system and height above sea level, focal length, yaw, 
 pitch and roll, dimensions of the camera sensor and resolution of aerial
 images,</li>
 <li>Reference surface, i.e. digital elevation model in the target coordinate
--- a/temporal/t.rast.algebra/t.rast.algebra.html
+++ b/temporal/t.rast.algebra/t.rast.algebra.html
@@ -33,7 +33,7 @@ assigned to STRDS or to the map lists re
 STRDS.
 <p>
 By default, only temporal topological relations among space time datasets
-(STDS) are evaluated. The <b>-s</b> flag can be used to aditionally
+(STDS) are evaluated. The <b>-s</b> flag can be used to additionally
 activate the evaluation of the spatial topology based on the spatial
 extent of maps.
 <p>
